installer/ansible/install_dependencies.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo Enabling universe repository
+sudo add-apt-repository universe
+
+echo Enabling docker repository
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+
+# Update local repositories
+echo Updating local repositories
+sudo apt-get update
+
+# Install ansible dependencies
+echo Installing dependencies
+sudo apt-get install -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common sshpass pv gawk
+
+# Install docker if not present
+if hash docker 2>/dev/null; then
+    echo Docker already installed, skipping.
+else
+    echo Installing docker
+    sudo apt-get install -y docker-ce docker-ce-cli containerd.io
+fi
+
+# Install docker compose
+echo Installing docker-compose
```

Comment:
No need to install docker-compose, we can use docker compose from the new install steps for docker

Reply:
currently supporting docker-compose instead of docker compose. @PravinRanjan10 pls let me know if any changes required here.

Reply:
okay. Please go ahead with docker compose (the new install steps)

Reply:
done